mcmcfit.py

```diff
@@ -224,9 +226,13 @@ def ln_prior(self,verbose=False):
                 sc = self.getParam(scaleTemplate.format(iecl))
                 e1 = self.getParam(exp1Template.format(iecl))
                 e2 = self.getParam(exp2Template.format(iecl))
-                if sc.currVal*(e1.currVal/e2.currVal)**(1.0/e2.currVal) > 0.5:
+            
+                if sc.currVal*(e1.currVal/e2.currVal) < 0. and e2.currVal < 0:
```

Comment:
This is not the right fix. The problem is that that BS flux is given by

![screenshot 2018-11-29 at 10 20 47](https://user-images.githubusercontent.com/4570807/49215321-719a2d00-f3c0-11e8-82ec-ea6d6748e5c9.png)

But sometimes (X/S) is negative and is raised to a fractional power. This needs an upstream fix in 
https://github.com/StuartLittlefair/lfit. Please raise an issue there.

In the meantime, please remove this addition